codemon/isModified.py

```diff
@@ -0,0 +1,13 @@
+import os
+from clint.textui import colored
+
+def isModified(event):

```

Comment:
This function can be added to `listen.py` as it's used over there.

Reply:
I think it would be great to divide the application in small fragments.

Reply:
@yashshah14093 - I agree its great to divide application into smaller fragments but functions which are related to each other should be on the same file. Since the functions are already quite short, we can definitely do that.